src/layouts/Main.astro

```diff
@@ -223,3 +251,19 @@ const githubEditUrl = `https://github.com/snowpackjs/astro-docs/blob/main/${curr
     </main>
   </body>
 </html>
+
+<script>
+    const btn = document.querySelector('#menu-toggle')
+    const sidebar = document.querySelector('#sidebar-nav')
+    let menuOpen = false
+    btn.addEventListener('click',()=>{
+      menuOpen = !menuOpen
+      if(menuOpen){
+        sidebar.style.display="block"
```

Comment:
Small lint thing, you're using double quotes here and single for none, I would opt for 1 choice

Reply:
😅 that one was a slip,